tribuo-classification/enclave/src/main/java/com/r3/conclave/sample/enclave/MlEnclave.java

```diff
@@ -97,18 +92,17 @@ protected void collectData(InputData inputData) {
 
     /**
      * This method deserializes input data from clients to be used by enclave.
-     * @param  inputData
+     * @param inputData
      * @return deserialized input data object
      */
-    public static Object deserialize(byte[] inputData) {
+    public static Object deserialize(byte[] inputData) throws RuntimeException {
```

Comment:
You don't need to add a throws declaration for unchecked exceptions. RuntimeException and its subclasses are example of unchecked exceptions.

Reply:
yes correct done.